core/tests/protractor_desktop/topicAndStoryViewer.js

```diff
@@ -161,11 +161,12 @@ describe('Topic and Story viewer functionality', function() {
 
     // Signing up with the login button should redirect the user back to the
     // exploration.
-    const loginButton = element(by.css('.protractor-test-login-button'));
-    await action.click('Login button', loginButton);
-    const useManualNavigation = false;
+    const LOGIN_BUTTON = element(by.css('.protractor-test-login-button'));
```

Comment:
@krishnarao22 Do we really need this style? I'm not sure about the requirement here, can you please check with the team and confirm?

Reply:
This style is not necessary, `const loginButton` is totally fine.

Reply:
@krishnarao22 So where are we expecting to have const VARIABLE_IN_CAPS? Can you please help me finalize the requirement? Thanks!

Reply:
@AdityaDubey0 Just to clarify, all constants should have the style `VARIABLE_IN_CAPS`. 

It's just that DOM elements shouldn't be constants -- they are variables.

For example, say we have a home button. In this case, we wouldn't have `const HOME_BUTTON`, we would have `var homeButton`.

TLDR: You don't have to change the linter, all constants should have the style `const VARIABLE_IN_CAPS`

Reply:
Thanks for info..